src/otx/v2/adapters/torch/mmengine/mmaction/dataset.py

```diff
@@ -0,0 +1,255 @@
+"""OTX adapters.torch.mmengine.mmaction.Dataset API."""
+
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+from __future__ import annotations
+
+from functools import partial
+from typing import Iterable
+
+import torch
+from mmengine.dataset import pseudo_collate, worker_init_fn
+from mmengine.dist import get_dist_info
+from mmengine.utils import digit_version
+from torch.utils.data import DataLoader as TorchDataLoader
+from torch.utils.data import Dataset as TorchDataset
+from torch.utils.data import Sampler
+
+from otx.v2.adapters.torch.mmengine.dataset import MMXDataset
+from otx.v2.adapters.torch.mmengine.mmaction.modules.datasets import (
+    OTXActionClsDataset,
+)
+from otx.v2.adapters.torch.mmengine.mmaction.registry import MMActionRegistry
+from otx.v2.api.entities.task_type import TaskType, TrainType
+from otx.v2.api.utils.decorators import add_subset_dataloader
+
+SUBSET_LIST = ["train", "val", "test", "predict"]
+
+
+def get_default_pipeline(subset: str) -> list:
+    """Returns the default pipeline for mmaction model.
+
+    Args:
+        subset (str): Subset of default pipeline
+
+    Returns:
+        List: The default pipeline as a dictionary or list, depending on whether `semisl` is True or False.
+    """
+    if subset not in SUBSET_LIST:
+        msg = f"{subset} is not supported subset"
+        raise ValueError(msg)
+
+    default_pipeline = {
+        "train": [
+            {
+                "type": "SampleFrames",
+                "clip_len": 8,
+                "frame_interval": 4,
+                "num_clips": 1,
+            },
+            {"type": "OTXRawFrameDecode"},
+            {"type": "Resize", "scale": (-1, 256)},
```

Comment:
Is train the only one that doesn't have a CenterCrop?

Reply:
Yes.